server.go

```diff
@@ -591,7 +592,20 @@ func (s *Server) HandleTracePacket(packet []byte) {
 		log.WithError(err).Warn("Trace unmarshaling error")
 		return
 	}
-
+	// fmt.Println(*newSample)
+	// fmt.Println(newSample.Metrics)
+	for _, metricPacket := range newSample.Metrics {
+		metric, err := samplers.ParseMetricSSF(metricPacket)
+		if err != nil {
+			log.WithFields(logrus.Fields{
+				logrus.ErrorKey: err,
+				"packet":        metricPacket.String(),
+			}).Warn("Could not parse packet")
+			s.Statsd.Count("packet.error_total", 1, []string{"packet_type:ssf_metric", "reason:parse"}, 1.0)
+			return
+		}
+		s.Workers[metric.Digest%uint32(len(s.Workers))].PacketChan <- *metric
+	}
 	s.TraceWorker.TraceChan <- *newSample
```

Comment:
It will likely be pretty common that that are "metrics only" arrive. Since everything is optional in protobuf, we'll need some way to say "is this span even a span?" and only send it to the trace worker if it is!